Gradle plugin not compatible with Gradle 5.0-rc-2

Gradle 5.0 has changed the API of the `SourceTask` class, which your `FormatTask` inherits. The api change made the `sources` field of `SourceTask` private, see https://github.com/gradle/gradle/commit/9e0f9be30d682882119f49ff5926d1e32ec26a27#diff-5267e6b16f7b156ab75dabc8537010ad.

When trying to run the task from Gradle, the following error appears:

```
class com.github.sherter.googlejavaformatgradleplugin.FormatTask tried to access private field org.gradle.api.tasks.SourceTask.source (com.github.sherter.googlejavaformatgradleplugin.FormatTask is in unnamed module of loader org.gradle.internal.classloader.VisitableURLClassLoader @376eac7a; org.gradle.api.tasks.SourceTask is in unnamed module of loader org.gradle.internal.classloader.VisitableURLClassLoader @4520ebad)
```

```
gradle --version                                                                                                                                                                      
------------------------------------------------------------
Gradle 5.0
------------------------------------------------------------

Build time:   2018-11-26 11:48:43 UTC
Revision:     7fc6e5abf2fc5fe0824aec8a0f5462664dbcd987

Kotlin DSL:   1.0.4
Kotlin:       1.3.10
Groovy:       2.5.4
Ant:          Apache Ant(TM) version 1.9.13 compiled on July 10 2018
JVM:          11.0.1 (Oracle Corporation 11.0.1+13-Ubuntu-2ubuntu1)
OS:           Linux 4.18.0-12-generic amd64
```

```
org.gradle.internal.event.ListenerNotificationException: Failed to notify task execution listener.
        at org.gradle.internal.event.AbstractBroadcastDispatch.dispatch(AbstractBroadcastDispatch.java:86)
        at org.gradle.internal.event.BroadcastDispatch$CompositeDispatch.dispatch(BroadcastDispatch.java:324)
        at org.gradle.internal.event.BroadcastDispatch$CompositeDispatch.dispatch(BroadcastDispatch.java:234)
        at org.gradle.internal.event.ListenerBroadcast.dispatch(ListenerBroadcast.java:140)
        at org.gradle.internal.event.ListenerBroadcast.dispatch(ListenerBroadcast.java:37)
        at org.gradle.internal.dispatch.ProxyDispatchAdapter$DispatchingInvocationHandler.invoke(ProxyDispatchAdapter.java:93)
        at com.sun.proxy.$Proxy104.beforeExecute(Unknown Source)
        at org.gradle.api.internal.tasks.execution.EventFiringTaskExecuter$1.run(EventFiringTaskExecuter.java:47)
        at org.gradle.internal.operations.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:301)
        at org.gradle.internal.operations.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:293)
        at org.gradle.internal.operations.DefaultBuildOperationExecutor.execute(DefaultBuildOperationExecutor.java:175)
        at org.gradle.internal.operations.DefaultBuildOperationExecutor.run(DefaultBuildOperationExecutor.java:91)
        at org.gradle.internal.operations.DelegatingBuildOperationExecutor.run(DelegatingBuildOperationExecutor.java:31)
        at org.gradle.api.internal.tasks.execution.EventFiringTaskExecuter.execute(EventFiringTaskExecuter.java:44)
        at org.gradle.execution.plan.LocalTaskNodeExecutor.execute(LocalTaskNodeExecutor.java:43)
        at org.gradle.execution.taskgraph.DefaultTaskExecutionGraph$InvokeNodeExecutorsAction.execute(DefaultTaskExecutionGraph.java:337)
        at org.gradle.execution.taskgraph.DefaultTaskExecutionGraph$InvokeNodeExecutorsAction.execute(DefaultTaskExecutionGraph.java:325)
        at org.gradle.execution.taskgraph.DefaultTaskExecutionGraph$BuildOperationAwareExecutionAction.execute(DefaultTaskExecutionGraph.java:318)
        at org.gradle.execution.taskgraph.DefaultTaskExecutionGraph$BuildOperationAwareExecutionAction.execute(DefaultTaskExecutionGraph.java:304)
        at org.gradle.execution.plan.DefaultPlanExecutor$ExecutorWorker$1.execute(DefaultPlanExecutor.java:134)
        at org.gradle.execution.plan.DefaultPlanExecutor$ExecutorWorker$1.execute(DefaultPlanExecutor.java:129)
        at org.gradle.execution.plan.DefaultPlanExecutor$ExecutorWorker.execute(DefaultPlanExecutor.java:202)
        at org.gradle.execution.plan.DefaultPlanExecutor$ExecutorWorker.executeNextNode(DefaultPlanExecutor.java:193)
        at org.gradle.execution.plan.DefaultPlanExecutor$ExecutorWorker.run(DefaultPlanExecutor.java:129)
        at org.gradle.internal.concurrent.ExecutorPolicy$CatchAndRecordFailures.onExecute(ExecutorPolicy.java:63)
        at org.gradle.internal.concurrent.ManagedExecutorImpl$1.run(ManagedExecutorImpl.java:46)
        at org.gradle.internal.concurrent.ThreadFactoryImpl$ManagedThreadRunnable.run(ThreadFactoryImpl.java:55)
Caused by: java.lang.IllegalAccessError: class com.github.sherter.googlejavaformatgradleplugin.FormatTask tried to access private field org.gradle.api.tasks.SourceTask.source (com.github.sherter.googlejavaformatgradleplugin.FormatTask is in unnamed module of loader org.gradle.internal.classloader.VisitableURLClassLoader @d492c41; org.gradle.api.tasks.SourceTask is in unnamed module of loader org.gradle.internal.classloader.VisitableURLClassLoader @20fa23c1)
        at com.github.sherter.googlejavaformatgradleplugin.FormatTask.hasSources(FormatTask.java:12)
        at com.github.sherter.googlejavaformatgradleplugin.TaskConfigurator.configureFormatTask(TaskConfigurator.java:22)
        at com.github.sherter.googlejavaformatgradleplugin.TaskConfigurator.configure(TaskConfigurator.java:39)
        at com.github.sherter.googlejavaformatgradleplugin.GoogleJavaFormat.accept(GoogleJavaFormat.groovy:24)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at com.github.sherter.googlejavaformatgradleplugin.GoogleJavaFormatPlugin$_apply_closure1.doCall(GoogleJavaFormatPlugin.groovy:41)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at org.gradle.listener.ClosureBackedMethodInvocationDispatch.dispatch(ClosureBackedMethodInvocationDispatch.java:40)
        at org.gradle.listener.ClosureBackedMethodInvocationDispatch.dispatch(ClosureBackedMethodInvocationDispatch.java:25)
        at org.gradle.internal.event.AbstractBroadcastDispatch.dispatch(AbstractBroadcastDispatch.java:42)
        at org.gradle.internal.event.BroadcastDispatch$SingletonDispatch.dispatch(BroadcastDispatch.java:230)
        at org.gradle.internal.event.BroadcastDispatch$SingletonDispatch.dispatch(BroadcastDispatch.java:149)
        at org.gradle.internal.event.AbstractBroadcastDispatch.dispatch(AbstractBroadcastDispatch.java:58)
        ... 26 more
```

--- a/src/main/groovy/com/github/sherter/googlejavaformatgradleplugin/FormatTask.java
+++ b/src/main/groovy/com/github/sherter/googlejavaformatgradleplugin/FormatTask.java
@@ -9,6 +9,6 @@ abstract class FormatTask extends SourceTask {
   abstract void accept(TaskConfigurator configurator);
 
   boolean hasSources() {
-    return !source.isEmpty();
+    return !getSource().isEmpty();
   }
 }

